bin/crew

```diff
@@ -1230,18 +1238,10 @@ def resolve_dependencies
 
   if @opt_force
     puts 'Proceeding with dependency package installation...'.orange
+  elsif agree('Do you agree? ')
```

Comment:
This feels like we're adding a large gem dependency and significantly increasing the dependencies and complexity of bin/crew in order to avoid 9 lines of code-- couldn't we just take the existing input prompt code we have and split it out into a file in `lib`, then call that?

Reply:
We could, but highline is so much better at asking for an answer than our code and makes our code simpler. Should we really be worried about loading an extra gem for a single threaded app on standalone systems, especially if it makes our code much more readable?

Look at it:

https://github.com/JEG2/highline

Reply:
Our code isn't being made overall simpler by having to work around a gem dependency-- If we were using it in tons of places for tons of things, sure, but we're only using it for an input prompt, which we already have the code for.

I just don't see the value in doing all this compared to splitting out the 10 lines of code into a helper file and calling that.

Reply:
I guess in part this is a stylistic question? I really don't like our prompt code. I think it is fragile and doesn't work well, and as such I'm happy to outsource to a gem that does it better.

The "I don't understand" bit there now is especially annoying because clearly there is code that can do a better job at understanding edge case inputs.

But of course I'm happy to let other people vote as to whether they think this is a better approach.

**My TL;DR here would be:** _It's ok if a slight increase in complexity much improves usability._

Reply:
We can (and probably should) potentially change to use `agree(...)` in tons of packages in the postremove block.  In agreement with @satmandu here.